pxr/usdImaging/usdImaging/cameraAdapter.cpp

```diff
@@ -259,9 +259,33 @@ UsdImagingCameraAdapter::Get(UsdPrim const& prim,
         cam.GetShutterCloseAttr().Get(&vShutterClose, time); // conversion n/a
         return vShutterClose;
     } else if (key == HdCameraTokens->exposure) {
-        VtValue v;
-        cam.GetExposureAttr().Get(&v, time); // conversion n/a
-        return v;
+        // we hijack the "exposure" token here in order to pass through the calculated
+        // exposure and make it backwards-compatible.
+        // existing clients will be expected it as logarithmic exposure, so we need
+        // to convert from a scalar multiplier to logarithmic exposure here.
+        // To get the original exposure attribute value, see "exposureCompensation"
+        // below
+        return VtValue(log2(cam.GetExposureScale(time)));
```

Comment:
We're missing the equivalent code in dataSourceCamera.cpp: https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/usdImaging/usdImaging/dataSourceCamera.cpp#L132

Reply:
(Note: I've implemented this internally).